### PR TITLE
Validation tests for labels

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

File: src/main/scala/centaur/test/standard/StandardTestCase.scala [hunk 1]
```diff
@@ -39,7 +39,7 @@ object StandardTestCase {
   def fromPath(path: Path): ErrorOr[StandardTestCase] = {
     Try(ConfigFactory.parseFile(path.toFile)) match {
       case Success(c) => StandardTestCase.fromConfig(c, path.getParent)
-      case Failure(f) => invalidNel(s"Invalid test config: $path")
+      case Failure(f) => invalidNel(s"Invalid test config: $path due to failure: $f")
```
Comment (danbills) [inline src/main/scala/centaur/test/standard/StandardTestCase.scala:42]: 👍

File: src/main/scala/centaur/test/workflow/WorkflowData.scala [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
-import configs.Result
+import configs.Result.{Failure, Success}
 import configs.syntax._
-import spray.json.JsArray
+import cromwell.api.model.Label
+import spray.json._
 
-case class WorkflowData(wdl: String, inputs: Option[String], options: Option[String], zippedImports: Option[File])
+
+case class WorkflowData(wdl: String, inputs: Option[String], options: Option[String], labels: Option[List[Label]], zippedImports: Option[File])
```
Comment (cjllanwarne) [inline src/main/scala/centaur/test/workflow/WorkflowData.scala]: I would say it's fine to just use `List[Label]` since that can already be empty.

**Review by cjllanwarne — APPROVED** [review]

**Comment by danbills** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/194/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by cjllanwarne** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/194/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**